packages/twenty-server/src/integrations/message-queue/drivers/bullmq.driver.ts

```diff
@@ -93,7 +93,11 @@ export class BullMQDriver implements MessageQueueDriver {
         `Queue ${queueName} is not registered, make sure you have added it as a queue provider`,
       );
     }
-    const queueOptions = { jobId: options?.id, priority: options?.priority };
+    const queueOptions = {
+      jobId: options?.id,
+      priority: options?.priority,
+      attempts: 1 + (options?.retryLimit || 0),
```

Comment:
is attempts only for bull-mq? why don't we have it in pg-boss? we should try to have both having the same interface.
also, retryAttempt ?

Reply:
They have the same QueueJobOptions interface @charlesBochet, they both use "retryLimit" in this case, however bull-mq uses "attempts" for this.